src/styles/Blocks/calendar.scss

```diff
@@ -0,0 +1,27 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  flex-wrap: wrap;
+  width: var(--calendar-width);
+
+  @each $day, $mar in mon 0,
+    tue 101px,wed 202px,
+    thu 303px,fri 404px,
+    sat 505px,sun 606px {
+    &--start-day_#{$day}>:first-child {
+      margin-left: $mar;
+    }
+  }
```

Comment:
$start-day:
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
  
    @each $day, $number in $start-day {
    &--start-#{$day} :first-child {
      margin-left: ($sixe-square + 1) * $number;
    }
  }